bridgerelayer/tools/list_events.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	gsrpc "github.com/snowfork/go-substrate-rpc-client"
+
+	"github.com/snowfork/go-substrate-rpc-client/types"
+)
+
+type Events []Event
+
+func (e Events) Len() int {
+	return len(e)
+}
+
+func (e Events) Less(i, j int) bool {
```

Comment:
Does it make sense to call this for clarity?

```suggestion
func (e Events) Lte(i, j int) bool {
```

Reply:
Ah `Less` is an Interface method which I'm required to implement so that I can sort an array of events.

Reply:
Okay, got it. Less would semantically be `<` though, not `<=`, right?